### PR TITLE
Fix view permissions small bugs #LMR-1193

### PR DESCRIPTION
**Pull request by kubedo8**

### CODE REVIEW

File: src/app/shared/preview-results/preview-results.component.ts [hunk 1]
```diff
@@ -143,7 +143,8 @@ export class PreviewResultsComponent implements OnInit, OnDestroy, OnChanges {
     const collectionQuery = {...this.query, collectionIds: [collection.id]};
     this.updateDataSubscription(collectionQuery);
     this.store.dispatch(new DocumentsAction.Get({query: collectionQuery}));
-    this.loaded$ = this.store.select(selectQueryDocumentsLoaded(collectionQuery));
+    this.allSubscriptions.add(this.store.select(selectQueryDocumentsLoaded(collectionQuery))
+      .subscribe(loaded => this.loaded$.next(loaded)));
```
Comment (livthomas) [inline src/app/shared/preview-results/preview-results.component.ts]: Why this change? The previous solution is better since you do not need to handle subscription outside of the template.

Comment (kubedo8) [reply src/app/shared/preview-results/preview-results.component.ts]: For some reason previous observable was not emitting values for the first time when you switch to detail perspective and I have no idea why.

Comment (livthomas) [reply src/app/shared/preview-results/preview-results.component.ts]: In such cases you can always use something like this:
```typescript
.pipe(startWith(false))
```

Comment (kubedo8) [reply src/app/shared/preview-results/preview-results.component.ts]: I will try it in the future, thanks :)
